PennMobile/General/Generics/GenericControllers.swift

```diff
@@ -106,3 +107,43 @@ class GenericViewController: UIViewController, Trackable {
         self.navigationItem.leftBarButtonItem = nil
     }
 }
+
+/// ``GenericViewController`` instance that uses a SwiftUI view for its content.
+///
+/// To use this view controller, subclass and override ``content`` and ``tabTitle``.
+class GenericSwiftUIViewController<Content: View>: GenericViewController {
+    /// Content of the view controller.
+    open var content: Content {
+        fatalError("makeSwiftUIView() not implemented")
```

Comment:
Nice 👍  We should use this for our dining view controller and future wrapped version of UIKit Controllers

Reply:
Yeah, was considering rewriting DiningViewController as well, but we could probably do that in a separate PR.

Reply:
Yup, we could complete it on another PR.